### DISCUSSION TITLE
Need Help Hotspots are not showing on my iOS devices

### DISCUSSION THREAD

**Discussion by sahimsakir** [google/model-viewer#3441] [General]
Hello, I have started newly on model viewer. Recently i was working on a project where I need to add hotspots which will change my  models by clicking the element. It's Working fine on browser and android devices but unfortunately i am not getting the hotspots on iOS. Can you help me to solve this?

link: https://artest.nvisio.one

**Comment by alexsafayan** [comment]
I am on Chrome on a MacBook Pro, and I don't see any hotspots?

**Reply by sahimsakir** [reply]
the hotspot will show on AR view

**Comment by samaneh-kazemi** [comment]
Hi @sahimsakir, thanks for reaching out. I am not fully following what doesn't work. I just opened the link in my iPhoneXR and was able to see the hotspot on safari. 

Can you please share more details on what device/browser you use and provide a video recording? 

https://user-images.githubusercontent.com/3915613/167678871-37353348-b2df-4bbc-91dd-23f58591097d.MOV

**Reply by sahimsakir** [reply]
hi @samaneh-kazemi ,
thank you for reaching out. Please press the AR button when the camera appear the hotspot will disappear on AR mode.

The Device is: Apple 13

**Reply by sahimsakir** [reply]
@samaneh-kazemi 
please check this below link
https://artest.nvisio.one/

**Reply by samaneh-kazemi** [reply]
hotspot is a HTML element. So you can only see it on the 3D model on browser and it won't work in AR mode in iOS.

**Reply by sahimsakir** [reply]
it's working on Andriod Devices but not working on iOS

**Reply by elalish** [reply]
If you can share images of what working and not working looks like, that will be very helpful. If you use WebXR (our default AR mode on Android) then HTML like hotspots will work. For Quick Look (the only option on iOS) or Scene Viewer, they are separate apps and any HTML/JS customization is lost.

**Reply by alexsafayan** [reply]
Quick Look is the only option on IOS? So there is no solution using model-viewer to have HTML elements in AR on IOS devices?

**Reply by elalish** [reply]
Sadly this is true until Apple implements WebXR in WebKit. Please let them know how you feel. @grorg for visibility.

**Reply by sahimsakir** [reply]
So what is the way to show hotspots in iOS AR?
